Library/Homebrew/api.rb

```diff
@@ -92,21 +92,62 @@ def self.fetch_json_api_file(endpoint, target:)
 
         retry
       end
+
+      if jws
+        success, data = verify_and_parse_jws(json_data)
+        unless success
+          target.unlink
+          odie "Failed to verify integrity of #{url} (#{data})."
```

Comment:
Probably want to tell people that we deleted the file and what, if any, action item there is?

Reply:
I guess `brew update` maybe? In theory this should only happen if there is MITM tampering going on or if they're on an ancient version of brew (if we ever roll a new key in the future). If people are ever seeing it then it's concerning.

Reply:
@Bo98 Yeh, I think something like "potential MITM attempt detected, please run `brew update` and try again"?